src/App.tsx

```diff
@@ -1,61 +1,135 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import cn from 'classnames';
+
+enum SortType {
+  SORT_ALPHABETICALLY = 'Sort alphabetically',
+  SORT_BY_LENGTH = 'Sort by length',
+  DEFAULT = '',
+}
 
 export const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
+  { id: 1, good: 'Dumplings' },
+  { id: 2, good: 'Carrot' },
+  { id: 3, good: 'Eggs' },
+  { id: 4, good: 'Ice cream' },
+  { id: 5, good: 'Apple' },
+  { id: 6, good: 'Bread' },
+  { id: 7, good: 'Fish' },
+  { id: 8, good: 'Honey' },
+  { id: 9, good: 'Jam' },
+  { id: 10, good: 'Garlic' },
 ];
 
+type Good = {
+  id: number;
+  good: string;
+};
+
+type Parametrs = {
+  sortFild: SortType;
+  reverseFild: boolean;
+};
+
+function getPreparedGoods(
+  goods: Good[],
+  { sortFild, reverseFild }: Parametrs,
+): Good[] {
+  const preparedGoods = [...goods];
+
+  if (sortFild) {
+    preparedGoods.sort(({ good: good1 }, { good: good2 }) => {
+      switch (sortFild) {
+        case SortType.SORT_ALPHABETICALLY:
+          return good1.localeCompare(good2);
+        case SortType.SORT_BY_LENGTH:
+          return good1.length - good2.length;
+        default:
```

Comment:
The default case in the sort switch statement returns 0. It would be better to return the original array in the default case instead of 0.